packages/app-form-builder/src/admin/components/FormEditor/Droppable.tsx

```diff
@@ -23,9 +23,13 @@ export interface DroppableCollectedProps {
 export interface IsVisibleCallableParams {
     type: string;
     isDragging: boolean;
-    ui: string;
-    pos?: Partial<FieldLayoutPositionType>;
-    formStep?: FbFormStep;
+    ui: "row" | "field" | "conditionGroup" | "step";
```

Comment:
Let's have a union type that we can reuse, and so that we don't have to write `"row" | "field" | "conditionGroup" | "step"` every time. We can probably add it it `~/types` file.